src/dos/dos_memory.cpp

```diff
@@ -18,23 +18,90 @@
 
 #include "dosbox.h"
 
-#include "mem.h"
 #include "dos_inc.h"
+#include "enum.h"
+#include "mem.h"
 
-#define UMB_START_SEG 0x9fff
+enum class McbFaultStrategy { deny, repair, report, allow };
```

Comment:
Don't we use always UpperCameCase for enum values?

Reply:
We have `enum.h` now, which gives us "enum reflection" to and from strings, amung other very nice enum features (it's the magic-enum header :magic_wand: :+1: ).

So in this case, it's a deliberate so we can cast directly from the conf string value (which will be lowercase) to the `enum class McbFaultStrategy` type.

Reply:
Ah okay, wasn't aware of that. Good to know!